docs/sources/tempo/operations/backend_search.md

```diff
@@ -141,7 +203,196 @@ querier:
+Instead of sending jobs one by one, batch the jobs and send them at once.
+The querier picks up and process them.
+
+Batching pushes jobs faster to the queriers, and reduce the time spent waiting around for jobs to be picked up by the querier.
+
+#### Guidelines
+
+* Default value of `max_batch_size` is set to `5`. In testing, `5` is a good default across all sizes of clusters.
+* We DO NOT recommend changing the batch size from the default value of `5`. In testing, `5` was the sweet spot.
+* If the batch size is big, you push more jobs at once but it takes longer for the querier to process the batch and return the results back.
+* Big batch size will increase the latency of the querier requests, and they might start hitting timeouts of 5xx, which will increase the rate of retries.
+* Bigger `max_batch_size` results in pushing too many jobs to the queriers. The jobs then have to be canceled if a query is exited early.
+
+###  `query_frontend.search.concurrent_jobs` parameter
+
+The number of concurrent jobs to execute when searching the backend. This controls how much work is produced concurrently.
+
+To put it another way, if a search job is sharded into 5000 jobs, and `concurrent_jobs` is set to 1000, then Tempo only executes 1000 jobs concurrently.
+The other 4000 jobs are processed one by one as the first 100 jobs return the responses.
+
+If Tempo manages to answer the search query without executing all 500 jobs, Tempo exits early and cancels the jobs that weren't started.
+
+#### Guidelines
+
+* This params controls the numbers of concurrent jobs to the number of jobs to put in the queue at once for processing a query.
+* If this number is low, big queries that create lots of jobs processed slowly.
+* If this number is high, the frontend pushes work faster to queries, and the frontend needs to cancel those jobs if the search exits early because the query is fulfilled.
+* If you see high load, spikes, or OOMs in the queriers, you can lower this configuration for a fair QoS and stability.
+* Having this number to a high value will allow a single tenant to push lots of work at once and overwhelm the querier. So keeping it to a lower number in a shared cluster allows for fair scheduling.
+* This parameter controls how fast jobs are pushed into the queue for queries to be picked up by queries
+* Adjusting this can impact your batch size so that metric should be watched while making changes to this configuration, and make sure that actual batch size is around the `max_batch_size`.
+* We recommend keeping it high in dedicated clusters with a single big tenant.
+* We recommend keeping it low in shared clusters because it ensures that a single tenant can't overwhelm the read path.
+
+
+### `query_frontend.max_retries` parameter
+
+This option controls the number of times to retry a request sent to a querier.
+Current configurations only retry if the return is 5xx (_translated from equivalent gRPC error_) from a querier.
```

Comment:
```suggestion
We only retry on 5xx from a queriers (_translated from equivalent gRPC error_).
```

Reply:
In our public docs, we try to refrain from using "we" unless it's in release notes.  I need to do another round on the doc to try to work through some of those.

Reply:
I'm going to ask on the docs-team channel to see what people think if it's okay to use "We" in the case of recommendations.  If it is, then I'll set a linter exception to ignore the "We" usage.